src/utils/clients.ts

```diff
@@ -63,12 +62,13 @@ const setDeviceState = (() => {
 			}
 
 			const eventType = targetState.connected ? 'connect' : 'disconnect';
-			return pooledPostAsync({
-				url: `https://${BALENA_API_HOST}/services/vpn/client-${eventType}`,
-				timeout: REQUEST_TIMEOUT,
-				form: _.extend({ service_id: service.getId() }, targetState),
-				headers: { Authorization: `Bearer ${apiKey}` },
-			})
+			return request
```

Comment:
Why is this no longer pooled?

Reply:
It is, it is just exported as `request` from `utils/index.ts`:

```ts
export { pooledRequest as request } from './request';
```